twilio_integration/twilio_integration/doctype/whatsapp_message/whatsapp_message.py

```diff
@@ -25,8 +33,8 @@ def send(self):
 			self.save(ignore_permissions=True)
 
 		except Exception as e:
-			self.db_set('status', "Error")
-			self.log_error(title=_('Twilio WhatsApp Message Error'), message=e)
+			self.db_set('status', "Failed")
+			self.log_error(title=_('Twilio WhatsApp Message Failed'), message=e)
```

Comment:
remove exception handling from here, do not use this method directly, always use the `send_one_wa_message` method instead, `send_one_wa_message` will catch the exception and set the relevant status itself